### DISCUSSION TITLE
Why does `mise uninstall go` not stop `mise up` from updating it?

### DISCUSSION THREAD

**Discussion by ylluminate** [jdx/mise#3629] [Q&A]
If we want to entirely yank out `go` from the clutches of `mise`, what's the correct way to do this? It seems that a `mise up` still pulls it down post facto.

It seems silly to have to do this:
```
edit ~/.config/mise/config.toml
# remove:
go = "latest"
```
To get this to stop...

**Comment by syhol** [comment]
I have great news for you @ylluminate! `mise unuse` is in the next release: https://github.com/jdx/mise/pull/3627. this will remove the tool from your config AND uninstall it. Also the existing aliases of `mise rm` and `mise remove` will move from `mise uninstall` to this new command.
